pkg/api/password.go

```diff
@@ -97,8 +99,12 @@ func (hs *HTTPServer) ResetPassword(c *contextmodel.ReqContext) response.Respons
 		return response.Error(http.StatusBadRequest, "Passwords do not match", nil)
 	}
 
-	password := user.Password(form.NewPassword)
-	if password.IsWeak() {
+	userPassword := user.Password(form.NewPassword)
+	if hs.Features.IsEnabled(c.Req.Context(), featuremgmt.FlagPasswordPolicy) {
```

Comment:
Not sure we need a feature toggle for this one. If the default value for for this config is false this will not be used and `ValidatePassword` will fallback to `.IsWeak` anyway.

Reply:
All new features have to be covered under a feature toggle. There are a lot of [benefits](https://github.com/grafana/hosted-grafana/wiki/All-Things-Feature-Toggles#why-feature-toggles) from having this feature under a toggle and it doesn't hurt to protect the implementation. Once this is approved, the feature toggle will be removed.

Reply:
Sorry I don't think I was clear enough. I don't see the need for a feature toggle at al. The config option already is a toggle and is false by default

Reply:
I removed the feature toggle